### DISCUSSION TITLE
How can I assign different content to the footer of different i18n translation pages?

### DISCUSSION THREAD

**Discussion by Lorde627** [facebook/docusaurus#9275] [Q&A]
I want to provide different footer links for different i18n pages. Currently, `footer.json` only allows for changing the corresponding labels, but not the `href` links. What I aim to achieve is for pages in different languages to display different content, such as:

## English Page
Walmart: https://walmart.com
Discord: https://Discord.com

## Japan Page
Google: https://Google.co.jp

**Comment by slorber** [comment]
You can use an env variable workaround to customize each localized site config, see https://github.com/facebook/docusaurus/issues/4542#issuecomment-1434839071
